Fmaraev/Hometask_1/palindrome.py

```diff
@@ -0,0 +1,6 @@
+x = str(input())
+a = x[::-1]
+if x == a:
```

Comment:
Please solve it using numbers operations as in this home task we told only about numeric operations.